schemas/IDLs/org.gel.models.cva.avro/1.0.0/Transactions.avdl

```diff
@@ -113,7 +122,7 @@ protocol TransactionProtocol {
         /**
         The assembly to which the variant refers
         */
-        org.gel.models.report.avro.Assembly assembly;
+        union {null, org.gel.models.report.avro.Assembly} assembly;
```

Comment:
Should this be moved to inside the data to avoid this?

Reply:
Not sure if I agree, but in any case this has multiple implications, would do in a different PR exclusively for this. If we remove it we lose the ability to separate which transactions belong to which assembly. Open to discussion